index.html

```diff
@@ -14,46 +14,190 @@
-    <header class="toolbar-section">
-      <div class="logo">
-        <a href="">My Logo</a>
-      </div>
-
-      <div>
-        <p><span class="iconify" data-icon="eva:menu-outline"></span></p>
-      </div>
-    </header>
-
-    <!-- main section -->
-    <main class="main-section">
-      <div class="content">
-        <p class="author">
-          I’m Ismail <br />
-          Glad to see <br />
-          you!
+    <!-------------------------------------- middle-section ---------------------------------------->
+
+    <section class="middle-section">
+      <!-- tonic section 1 -->
+      <section class="tonic-section">
+        <div class="tonic-image"></div>
+        <h2 class="tonic">Tonic</h2>
+        <div class="canopy-container">
+          <p>CANOPY</p>
+          <p class="date">• Back End Dev • 2015</p>
+        </div>
+        <p class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="button-container">
+          <li><button class="html">html</button></li>
+          <li><button class="css">css</button></li>
+          <li><button class="javascript">javascript</button></li>
+        </ul>
+
+        <div class="see-project">
+          <button class="btn">See Project</button>
+        </div>
+      </section>
+
+      <!-- multipurpose section1 -->
+      <section class="multipurpose-section">
+        <div class="multipurpose-image"></div>
+
+        <h2 class="tonic">Tonic</h2>
+        <div class="canopy-container">
+          <p>CANOPY</p>
+          <p class="date">• Back End Dev • 2015</p>
+        </div>
+        <p class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="button-container">
+          <li><button class="html">html</button></li>
+          <li><button class="css">css</button></li>
+          <li><button class="javascript">javascript</button></li>
+        </ul>
+
+        <div class="see-project">
+          <button class="btn">See Project</button>
+        </div>
+      </section>
+
+      <!-- multipost section -->
+      <section class="post-section">
+        <div class="postsection-image"></div>
+
+        <h2 class="tonic">Tonic</h2>
+        <div class="canopy-container">
+          <p>CANOPY</p>
+          <p class="date">• Back End Dev • 2015</p>
+        </div>
+        <p class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="button-container">
+          <li><button class="html">html</button></li>
+          <li><button class="css">css</button></li>
+          <li><button class="javascript">javascript</button></li>
+        </ul>
+
+        <div class="see-project">
+          <button class="btn">See Project</button>
+        </div>
+      </section>
```

Comment:
In the project requirements, each card section should be styled using `flexbox`. It would be nice if you implemented this. 🙏